app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -77,6 +78,10 @@ def emails
       emails
     end
 
+    def proofing_rate_report
+      @proofing_rate_report ||= Reporting::ProofingRateReport.new(end_date: report_date)
+    end
+
     def account_reuse_report
       @account_reuse_report ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
```

Comment:
*Not* for this PR, but -- it is starting to feel really dirty that `Reporting::AccountReuseAndTotalIdentitiesReport` (and most other reports here) don't require an, erm, `require`, because they're local, but `Reporting::ProofingRateReport` does because it's in a _separate_ reporting directory. I hope some day we can unify that.